unwebpack_sourcemap.py

```diff
@@ -209,6 +218,7 @@ def _get_sanitised_file_path(self, sourcePath):
 
     def _get_remote_data(self, uri):
         """Get remote data via http."""
+        print(f"fetching {uri}")
```

Comment:
Pedantry: Can we make this line 

```
print("Fetching %s..." % uri)
```

in order to match the style used elsewhere in the code?